### PR TITLE
The GLORIOUS Supplypod Renaissance PART 2: The COOL GRAPHICS Update [EPIC MEME VIDEO INSIDE]

### PR DESCRIPTION
**Pull request by MrDoomBringer**
# [Please check out this EPIC OFFICIAL PR TRAILER before reading (very important)](https://youtu.be/4RckVdx20oQ)
the video is kinda autistic

dont forget to upvote
## About The Pull Request

All sprites and epic meme videos made with love by yours truly
<sup>Except for the 2 missile sprites. Those were taken and modified from NSV13 who in turn ported them from TGMC</sup>
nupod_all_decals

Anyways, this adds:
New pod, reverse-pod, and target indicator sprites
Pod-ground collision effects (impact craters basically)
Cool looking vapor trails a la the half-life 2 headcrab shells 

And also:
Fixes seethrough pods not actually working
Makes reverse-mode a bit more predictable
Has some minor code improvements with how supplypod-reverse-mode works

## Why It's Good For The Game
Yall really let #33436 get merged with the pods looking like that?

## Changelog
:cl:  MrDoomBringer
imageadd: Nanotrasen's Transportation and Logistics Subdivision has spent the better part of their budget developing a brand new model of Supplypod. Internal investigations have revealed the only real differences are almost purely aesthetic.
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
Note that supplypod code has gotten pretty bloated, next time on Part 3 we'll refactor all this nastiness.

**Comment by optimumtact** [comment]
I swear to god every time you make a PR your powerlevel increases

**Comment by kopoba** [comment]
can we have sindi red and clown pink?

**Comment by MrDoomBringer** [comment]
@kopoba garner some upthumbs and i'll make it so

**Comment by zxaber** [comment]
>Fixes seethrough pods not actually working

Wait, what? I use them all the time and they seem fine enough. What was the issue?

**Comment by PKPenguin321** [comment]
you should resprite the gondola pod so the opened door is actually its open mouth with its lower jaw dropping really really low

**Comment by MrDoomBringer** [comment]
@zxaber From what I could tell they didn't seem to work properly with mobs

**Comment by zxaber** [comment]
Oh, I suppose I haven't really tried with mobs. If that wasn't working, good catch.

**Comment by GravitronBlue** [comment]
I'd like to see the bluespace pod keep the white paint it had previously.

**Comment by MrDoomBringer** [comment]
@GravitronBlue I tried experimenting with that for a while but it never came out quite right, I'll give it another stab in a bit and post it here to see if you like it

**Comment by Doosteroni** [comment]
There should be like a skeleton in the blood cult pod

**Comment by Ryll-Ryll** [comment]
Hypest trailer 2020

**Comment by Dorsisdwarf** [comment]
I agree that I'm not sure if I like the loss of all color differentiation leaving all standard supply pods as just "choose a decal" on identical bases, but these sprites are VERY nice.

**Comment by MrDoomBringer** [comment]
Thoughts on something like this?
image
Shoutout to @davethwave for the syndicate pod sprite!

**Comment by GoldenAlpharex** [comment]
That looks amazing! Just one thing, though: Would it possible to also have the old sprites still available? That way, we could also still mess around with those too, for people that liked what they looked like. Otherwise, good job, this is just insane.

**Comment by MrDoomBringer** [comment]
@GoldenAlpharex whew why not

**Comment by Bastian0930** [comment]
litterally the video made me laugh

please make more over the top videos and ping me when done with them

it's fucking hilarious

**Comment by SweptWasTaken** [comment]
Did you spend more time on the video or the pr?

(I approve of both, well done)

File: code/__DEFINES/cargo.dm [hunk 1]
```diff
@@ -13,23 +13,30 @@
...
+    list(FALSE, "cultist",			RUBBLE_NORMAL,	"bloody supply pod", 				"A Nanotrasen supply pod covered in scratch-marks, blood, and strange runes."),\
+    list(TRUE, "missile",			RUBBLE_THIN,	"cruise missile", 						"A big ass missile that didn't seem to fully detonate. It was likely launched from some far-off deep space missile silo. There appears to be an auxillery payload hatch on the side, though manually opening it is likely impossible."),\
+    list(TRUE, "syndicatemissile",	RUBBLE_THIN,	"\improper Syndicate cruise missile", 	"A big ass, blood-red missile that didn't seem to fully detonate. It was likely launched from some deep space Syndicate missile silo. There appears to be an auxillery payload hatch on the side, though manually opening it is likely impossible."),\
+    list(TRUE, "box",				RUBBLE_WIDE,	"\improper Aussec supply crate", 		"An incredibly sturdy supply crate, designed to withstand orbital re-entry. Has 'Aussec Armory - 2532' engraved on the side."),\
+    list(TRUE, "clown", "clown",	RUBBLE_NORMAL,	"\improper HONK pod", 				"A brightly-colored supply pod. It likely originated from the Clown Federation."),\
```
Comment (Fikou) [inline code/__DEFINES/cargo.dm:37]: you put in "clown" twice

**Comment by MrDoomBringer** [comment]
image
slightly more hi-def base sprites also comin in

**Comment by Moja10** [comment]
MrDoomBringer is the new oranges

**Comment by MrHorizons** [comment]
> MrDoomBringer is the new oranges

I'll believe it when he removes cloning again.

**Comment by MrDoomBringer** [comment]
image
cool new effects booster pack just added

**Comment by MemedHams** [comment]
Quality animation in YOUR spessman game?

It's more likely than you think.

**Review by kriskog — DISMISSED** [review]
Fails the linter.
```
file ./code/modules/cargo/supplypod.dm is missing a trailing newline
```

**Comment by GoldenAlpharex** [comment]
Also, I don't know if that issue has been addressed with this PR, but just to be sure: Do you think you could remove the Alert Ghost option from being turned on by default? It's kind of annoying, as we almost never actually want it on.

**Comment by kriskog** [comment]
>  It's kind of annoying, as we almost never actually want it on.

Why would you not want this on? Its there to entertain bored ghosts.

**Comment by GoldenAlpharex** [comment]
> > It's kind of annoying, as we almost never actually want it on.
> 
> Why would you not want this on? Its there to entertain bored ghosts.

Because I mostly use it for uninteresting stuff.

**Comment by MrDoomBringer** [comment]
@GoldenAlpharex Soon(tm) i plan to add save slots (of which you can choose a default) for the supplypod menu, so that should solve your issue

**Comment by MrDoomBringer** [comment]
linter has been quelled

**Comment by yeeyeh** [comment]
The trailer alone is good enough to sell me on this. 10/10 beautiful execution

**Review by Twaticus — APPROVED** [review]
:heart:

**Pull request merged**